torch/csrc/jit/fuser/codegen.cpp

```diff
@@ -420,11 +368,10 @@ generateKernel(
   }
```

Comment:
On line 362: has_random isn't used anymore and can be deleted

Reply:
It's checked on line 397 to see if we should include RandHeader, RandParam, and RandInit.

Reply:
Oh I see, thanks for pointing that out!